OmiseSDK/Client.swift

```diff
@@ -138,6 +138,113 @@ import os
         })
         dataTask.resume()
     }
+    
+    public func retrieveChargeStatusWithCompletionHandler(from tokenID: String, completionHandler: ((Result<ChargeStatus, Error>) -> Void)?) {
```

Comment:
Is there a reason you made the `completionHandler` optional?

Reply:
Cause in this case I didn't use an `@escaping` then I make an optional `closure`. 
So, what do you think if we change to
`func retrieveChargeStatusWithCompletionHandler(from tokenID: String, completionHandler: @escaping ((Result<ChargeStatus, Error>) -> Void))` instead?

note: [optional-closure-are-escaping](https://www.jessesquires.com/blog/2018/06/10/why-optional-swift-closures-are-escaping/)

Reply:
Looks good. 👍

Reply:
Updated 🚀